### DISCUSSION TITLE
Multi Sensor Camera (Galaxy S23 Ultra)

### DISCUSSION THREAD

**Discussion by lzinga** [alicevision/Meshroom#2043] [Q&A]
I am not super familiar with cameras and am trying to add my phones sensor to the database. It is a Samsung Galaxy S23 Ultra. I found it on the device specifications but I am un sure what to use or how the camera sensor would be entered into the database correctly.

https://www.devicespecifications.com/en/model/1d8c5b9a

**Comment by natowi** [comment]
You should use the information from the main rear camera. The others (ultra-wide, telephoto1/2) are not the ones you would typically use for photogrammetry.

**Reply by lzinga** [reply]
Now I may be blind or may not be looking at the correct sections, but the page doesn't seem to have the main rear camera sensor info. It has labeled out cameras 2, 3 and 4 but not the main one it seems.

**Reply by natowi** [reply]
It is the camera listed under rear camera with 1/1.3"
